build_projects/dotnet-cli-build/PkgTargets.cs

```diff
@@ -45,10 +46,19 @@ public static BuildTargetResult InitPkg(BuildTargetContext c)
             return c.Success();
         }
 
+        public override bool Execute()
+        {
+            BuildContext context = new BuildSetup("MSBuild").UseAllTargetsFromAssembly<DebTargets>().CreateBuildContext();
+            BuildTargetContext c = new BuildTargetContext(context, null, null);
+
+            return GeneratePkgs(c).Success;
+        }
+
         public static BuildTargetResult GeneratePkgs(BuildTargetContext c)
         {
             if (CurrentPlatform.IsPlatform(BuildPlatform.OSX))
             {
+                PrepareTargets.Init(c);
```

Comment:
What from Init is necessary?  Can we change it over to use the MSBuild Init properties instead, now that those are done?